agentMET4FOF/metrological_streams.py

```diff
@@ -49,8 +49,10 @@ class MetrologicalDataStreamMET4FOF(DataStreamMET4FOF):
 
     def __init__(
         self,
-        value_unc: Union[float, Iterable[float]] = 0.0,
-        time_unc: Union[float, Iterable[float]] = 0.0,
+        value_unc: float = 0.0,
+        time_unc: float = 0.0,
+        exp_unc: float = None,
+        cov_factor: int = 1,
```

Comment:
We should include the Optional type hint as well.
 
```suggestion
        value_unc: Optional[float] = 0.0,
        time_unc: Optional[float] = 0.0,
        exp_unc: Optional[float] = None,
        cov_factor: Optional[int] = 1,
```